pkg/bindings/connection.go

```diff
@@ -210,15 +209,27 @@ func sshClient(_url *url.URL, uri string, identity string, machine bool) (Connec
 
 		if identity == "" {
 			if val := cfg.Get(alias, "IdentityFile"); val != "" {
+				// we get default IdentityFile value (~/.ssh/identity) every time
+				// checking if we got default
+				defaultIdentityPath := val == ssh_config.Default(strings.ToLower("IdentityFile"))
```

Comment:
you do not need to call ToLower(), the Default() function already does this so it is wasted.

Reply:
ok, let me fix that

Reply:
done